src/main/java/com/manardenza/console/ContentsUserMenu.java

```diff
@@ -0,0 +1,346 @@
+                break;
+            case (2):
+                outputDataInConsole("\tBuy!");
+                System.exit(0);
+                break;
+            default:
+                outputDataInConsole("Incorrect menu item");
+                mainMenu();
+                break;
+        }
+    }
+
+    public void serviceMenu() throws IOException {
+        boolean flagMenu = true;
+        while (flagMenu) {
+            outputSplitLine();
+            outputDataInConsole("\n\tService menu");
+            showUserMenu("Service menu");
+            int actionService = readerMenuFromConsole("Choose action: ");
+            switch (actionService) {
+                case (1):
+                    outputSplitLine();
+                    outputDataInConsole("For find hotel by name complete the following form");
+                    findHotelByNameMenu();
+                    outputSplitLine();
+                    break;
+                case (2):
+                    outputSplitLine();
+                    outputDataInConsole("For find hotel by city complete the following form");
+                    outputDataInConsole(findHotelByCityMenu().toString());
+                    outputSplitLine();
+                    break;
+                case (3):
+                    outputSplitLine();
+                    outputDataInConsole("To find the room fill in the following form");
+                    findRoomMenu();
+                    outputSplitLine();
+                    break;
+                case (4):
+                    outputSplitLine();
+                    outputDataInConsole("Your reserved rooms: ");
+                    getAllUserReservationsMenu();
+                    break;
+                case (5):
+                    outputSplitLine();
+                    outputDataInConsole("To cancel a reservation of the room, fill in the following form");
+                    cancelReservationMenu();
+                    outputSplitLine();
+                    break;
+                case (6):
+                    outputSplitLine();
+                    outputDataInConsole("\tThank you for using our service.");
+                    outputSplitLine();
+                    flagMenu = false;
+                    logoutUserMenu();
+                    continue;
+                default:
+                    outputDataInConsole("Incorrect menu item");
+                    serviceMenu();
+                    continue;
+            }
+        }
+    }
+
+    private void registrationUserMenu() {
+        String firstName = null;
+        String lastName = null;
+        outputSplitLine();
+        try {
+            firstName = String.valueOf(inputDataFromUser("Enter name: ", false,
+                    false, true, false));
+            lastName = String.valueOf(inputDataFromUser("Enter last name: ", false,
+                    false, true, false));
+            userController.loginUser(firstName, lastName);
+        } catch (IOException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1166)